docs/configuration.md

```diff
@@ -30,6 +30,10 @@ by passing the `--local` option to the `config` command.
 poetry config virtualenvs.create false --local
 ```
 
+Your local configuration of Poetry application is stored in `poetry.toml` file,
+which is separate from `pyproject.toml`.
+You should not check-in this file into your repository.
```

Comment:
not sure if thats true. I would include this into repo.

Reply:
@finswimmer has written in https://github.com/python-poetry/poetry/issues/2937#issuecomment-696312341 :

> The `pyproject.toml` contains information about your package, like metadata and dependencies. Anything that is needed to build the package.
> 
> The `poetry.toml` contains the local configuration of the poetry application. One should not check-in this file into a repository, because these configuration are just personal flavors.

Should I use different wording if this is not *always* true?

Reply:
oh, didn't know about that. Nevermind then.